src/lang/en-US.ts

```diff
@@ -336,8 +336,8 @@ const AdminPage: ResolvedIntlConfig['messages'] = {
     'admin.roles.message': `These are all the ${CommonMessages['terms.permissions']} that are currently provisioned. An administrator can update them in the {docLink}.`,
     'admin.roles.message.docLink': `authorization settings`,
     'admin.roles.message.docPath': `https://go.estuary.dev/provision`,
-    'admin.accessToken': `Access Token`,
-    'admin.accessToken.message': `Copy the access token below to authenticate the CLI client.`,
+    'admin.accessToken': `Service Account`,
+    'admin.accessToken.message': `Copy the command below to authenticate the CLI client.`,
```

Comment:
Since we are now giving the user a command to run in the terminal I think we need to include more details or at the very least link to the docs page: https://docs.estuary.dev/reference/authentication/#authenticating-flow-using-the-cli

Also... we'll need to update the docs page when we merge this.